modules/dist/localstorage.dev.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.getFromLocalStorage = exports.addToLocalStorage = void 0;
+
+var _index = require("./index.js");
+
+//  eslint-disable-next-line import/no-cycle
+var addToLocalStorage = function addToLocalStorage(books) {
+  window.localStorage.setItem('books', JSON.stringify(books));
+};
+
+exports.addToLocalStorage = addToLocalStorage;
+
+var getFromLocalStorage = function getFromLocalStorage() {
+  if (localStorage.length !== 0) {
+    var booksFromLocStg = JSON.parse(window.localStorage.getItem('books'));
+    booksFromLocStg.bookList.forEach(function (book) {
+      _index.books.bookList.push(book);
```

Comment:
- [ ] according to the requirements you should only use ES6 syntax, which includes arrow functions over the use of the function keyword, please adjust this.